src/app/containers/Timestamp/index.jsx

```diff
@@ -13,18 +13,31 @@ import {
 } from './timestampUtilities';
 import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 
-const isToday = timestamp => {
-  const today = moment(Date.now());
-  return today.isSame(timestamp, 'day');
+const isSameDay = (dayToCompare, timestamp) => {
+  const day = moment(dayToCompare);
+  return day.isSame(timestamp, 'day');
 };
 
-const formatType = timestamp =>
-  isToday(timestamp) ? formatDateAndTime : formatDate;
+const isToday = timestamp => isSameDay(Date.now(), timestamp);
 
-const humanReadable = ({ timestamp, shouldMakeRelative }) =>
+const formatType = timestamps => {
```

Comment:
This function is a tad confusing IMO, im not certain what it does 🤔 Could you maybe do something like
```
const formatType = ([ foo, bar ]) => {
```
Where the names give some indication of what its doing, atm I cant tell